code/modules/shuttle/emergency.dm

```diff
@@ -452,7 +452,7 @@
 					destination_dock = "emergency_syndicate"
 					minor_announce("Corruption detected in \
 						shuttle navigation protocols. Please contact your \
-						supervisor.", "SYSTEM ERROR:", alert=TRUE, custom_alert_sound = 'sound/misc/announce2.ogg')
+						supervisor.", "SYSTEM ERROR:", alert=TRUE)
```

Comment:
is this sound playing moved somewhere else?

Reply:
readded this back